lib/services/nodes-api-service.js

```diff
@@ -316,6 +314,9 @@ function nodeApiServiceFactory(
                 };
                 return workflowApiService.createAndRunGraph(configuration);
             }
+            else if(node.type === Constants.NodeTypes.Rack) {
+                return eventsProtocol.publishNodeEvent(node, 'added');
```

Comment:
@VulpesArtificem I think keep it for all types of nodes are good than just meet the 'rack' node requirement, because current codes has cover the 'rack' related event requirement， so I suggest close this PR, for docs, I'm submitting a PR for event refactoring, I can modify the docs there. you don't need to submit another PR for docs